docker/Dockerfile

```diff
@@ -70,12 +70,6 @@ RUN \
   curl -sL -o /opt/mozdef/nodesource.rpm https://rpm.nodesource.com/pub_4.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm && \
   rpm -i --nosignature --force /opt/mozdef/nodesource.rpm && \
   yum install -y nodejs-$NODE_VERSION && \
-  npm install source-map-support@0.4.2 \
```

Comment:
I moved this list of dependencies and stored them in https://github.com/mozilla/MozDef/pull/538/files#diff-d7366c300fa0f1d70a1278125050d8d7R10, so that we only need to do "meteor npm install". If/when we need to update the node libraries we use, we'll only need to modify the file, and not change any of the deployment.